fastlane/Fastfile

```diff
@@ -477,11 +510,11 @@ REPOSITORY_NAME="WordPress-Android"
     download_path=Dir.pwd + "/metadata/android"
     gp_downloadmetadata(project_url: "https://translate.wordpress.org/projects/apps/android/release-notes/",
       target_files: files,
-      locales: SUPPORTED_LOCALES.map {| hsh | [ hsh[:glotpress], hsh[:google_play] ]},
+      locales: RELEASE_NOTES_LOCALES.map {| hsh | [ hsh[:glotpress], hsh[:google_play] ]},
```

Comment:
The `.map` part was already applied when you created the `RELEASE_NOTES_LOCALES` constant.

You should thus either remove the `.map` from the definition of `RELEASE_NOTES_LOCALES` on line 67, or keep it on line 67 but remove it from this line 513. Given it's used in the exact same way on line 517 below too, I suggest the former solution, so that we only do the `.map` once.

```suggestion
      locales: RELEASE_NOTES_LOCALES,
```